modules/ROOT/pages/vpn-maintenance.adoc

```diff
@@ -0,0 +1,89 @@
+
+=== Tunnel is Going Down Due to Inactivity
+
+If there is no traffic over the VPN tunnel, it may go down due to inactivity. You can generate interesting traffic, or alternatively you can set Anypoint VPN to be the initiator. See https://help.mulesoft.com/s/article/How-to-enable-Initiator-mode-and-change-DPD-action-in-Anypoint-VPN[How to Enable Initiator Mode and Change DPD Action in Anypoint VPN^] for more information.
+
+=== IKE Phase Lifetimes Mismatch
+
+==== If you are using IKEv2:
+
+To maintain reliable tunnel operation after the upgrade, ensure that IKEv2 Phase 1 and Phase 2 lifetimes are set as described in xref:vpn-about.adoc#ipsec-settings[the documentation^]:
+
+* IKE Phase 1 lifetime: 28000 seconds
+* IKE Phase 2 lifetime: 3000 second
+
+Perform the upgrade of your non-production Anypoint VPNs first and ensure the tunnel is up, then plan for upgrading your Production Anypoint VPNs.
+
+[NOTE]
+AWS documentation states 28800 seconds for IKE Phase 1 lifetime, and 3600 seconds for IKE Phase 2 lifetime. We suggest the 28000/3000 combination for your firewall device to timeout first and control the rekey process.
+
+==== If you are using IKEv1:
+
+If you are using IKEv1 please use the default configuration: 28800 seconds for Phase 1 lifetime, and 3600 seconds for Phase 2 lifetime.
+
+=== Anypoint VPN Goes Down Periodically Due to PFS not Enabled for IPsec Phase 2
+
+If you are using Azure VPN, ensure that PFS is enabled and adjust lifetimes for phase 2. For more information, see:
+
+* https://help.mulesoft.com/s/article/Anypoint-VPN-goes-Down-Periodically-Due-to-PFS-is-not-Enabled-for-IPsec-Phase-2[Anypoint VPN Goes Down Periodically Due to PFS not Enabled for IPsec Phase 2^]
+* https://help.mulesoft.com/s/article/Anypoint-VPN-to-Azure-keeps-going-down-every-hour[Anypoint VPN to Azure Keeps Going Down Every Hour^]
+
+=== Avoid Concurrent VPN Updates
+
+To prevent VPN update errors, update only one VPN at a time. If you update multiple VPNs at once, the call may fail and you will need to manually reset the update flag.
+
+=== Anypoint VPN with BGP Routing Enters Asymmetric Mode after Update
+
+Anypoint VPN enters into asymmetric model after an update if the AS paths for the two tunnels of the same VPN are different. See https://help.mulesoft.com/s/article/Anypoint-VPN-with-BGP-Routing-Enters-Asymmetric-Mode-After-update[Anypoint VPN with BGP Routing Enters Asymmetric Mode after Upgrade^] for more information.
+
+=== Seeing IPsec DPD Failures in VPN Logs
+
+Ensure your DPD settings are correctly configured. See https://help.mulesoft.com/s/article/Seeing-IPsec-DPD-Failures-in-VPN-Logs[Seeing IPsec DPD Failures in VPN Logs^] for more information.
+
+=== Other issues
+
+For other issues with Anypoint VPN not particularly related to the update itself, see  the general troubleshooting xref:vpn-troubleshooting.adoc[documentation^].
```

Comment:
I see ^ at the end. Is this required?

Reply:
Just checked and It's not in this case (link to internal page of our Docs), but only when to linking external sites. I'll remove those that are not necessary.